From jaswinder@kernel.org  Thu Aug 13 16:41:24 2009
From: Jaswinder Singh Rajput <jaswinder@kernel.org>
Date: Thu, 13 Aug 2009 13:56:20 +0530
Subject: tty: includecheck fix: drivers/char, vt.c
To: Andrew Morton <akpm@linux-foundation.org>, Alan Cox <alan@lxorguk.ukuu.org.uk>, Greg KH <greg@kroah.com>
Message-ID: <1250151980.6116.18.camel@ht.satnam>

From: Jaswinder Singh Rajput <jaswinder@kernel.org>

fix the following 'make includecheck' warning:

  drivers/char/vt.c: linux/device.h is included more than once.

Signed-off-by: Jaswinder Singh Rajput <jaswinderrajput@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/char/vt.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/char/vt.c
+++ b/drivers/char/vt.c
@@ -2958,7 +2958,6 @@ int __init vty_init(const struct file_op
 }
 
 #ifndef VT_SINGLE_DRIVER
-#include <linux/device.h>
 
 static struct class *vtconsole_class;
 
